index.html

```diff
@@ -4158,6 +4169,60 @@ <h2><span class="chunk">cLLi</span> Content Light Level Information</h2>
         </section>
       </section>
 
+      <section id="dWLm-chunk">
+        <h2><span class="chunk">dWLm</span> Diffuse white luminance metadata</h2>
+        <p>The two-byte chunk type field contains the hexadecimal values</p>
+
+        <pre>
+<!-- xx xx xx xx
+</pre>
+
+        <p>If present, the <span class="chunk">dWLm</span> chunk stores the luminance value 
+          of the images nominal diffuse white, in cd/m2 as defined in [ISO/TS 22028-5]:</p>
+
+        <p>If the luminance of the diffuse white differs from the nominal luminance for an HDR reference displays
+          diffuse white as defined in in [[ITU-R-BT.2408]], the contents diffuse white luminance level
+          should be indicated using this metadata.</p>
+           
+         <p>The following specifies the syntax of the <span class="chunk">dWLn</span> chunk:</p>
+
+         <table id="dWLn-chunk-syntax" class="numbered simple">
+           <caption>dWLn chunk components</caption>
+
+           <tr>
+             <th>Name</th>
+             <th>Size</th>
+             <th>Divisor value</th>
+           </tr>
+
+           <tr>
+             <td>Diffuse white luminance metadata</td>
+             <td>2 bytes</td>
+             <td>1 cd/m<sup>2</sup></td>
+           </tr>
+
+         </table>
+
+         <aside class="example">
+           Example <span class="chunk">dWLn</span> Diffuse white luminance metadata:
+
+           <table id="dWLn-diffuse-white-luminance-metadata-example" class="numbered simple">
+             <tr>
+               <th>Actual value</th>
+               <th>Stored Decimal value</th>
+               <th>Stored 2-byte unsigned value</th>
+             </tr>
+
+             <tr>
+               <td>203 cd/m<sup>2</sup></td>
```

Comment:
203 is a poor example because this metadata is to be used when the diffuse white luminance is **not** 203!

```suggestion
               <td>100 cd/m<sup>2</sup></td>
```